application/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -32,6 +35,8 @@ import java.util.concurrent.TimeUnit
 
 object Server extends IOApp.WithContext {
 
+  implicit def unsafeLogger[F[_]: Sync] = Slf4jLogger.getLogger[F]
```

Comment:
Hm, so you need it for the `CollectionsService`, let's remove this implicit, it may cause issues and hard to understand what function requires it. Let's create an instance of it in the for comprehention via the `implicit0`, below?

Reply:
2e5928f